src/entities/Ticket.java

```diff
@@ -39,16 +40,16 @@ public void calculatePrice() {
     	//gets price from prices database and does algorithm on it.
     	//prices database stores default values for this particular movie and cinema and showtime
     	//if child or senior x0.8
-    	int moviePrice;
+    	double moviePrice;
     	MovieType movieType;
     	int customerAge;
-	SeatType seatType;
+    	SeatType seatType;
```

Comment:
Might need to look at the access modifier for enum in Seat class again. Since it is private, classes outside of Seat is unable to reference to it.